testsuite/features/core/proxy_branch_network.feature

```diff
@@ -31,15 +31,15 @@ Feature: Setup Uyuni for Retail branch network
 @proxy
 @private_net
 @susemanager
-  Scenario: Install the Retail pattern on the server
+  Scenario: Install the Retail pattern on the SUSE Manager server
     When I refresh the metadata for "server"
     When I install pattern "suma_retail" on this "server"
     And I wait for "patterns-suma_retail" to be installed on "server"
 
 @proxy
 @private_net
 @uyuni
-  Scenario: Install the Retail pattern on the server
+  Scenario: Install the Retail pattern on the Uyuni server
```

Comment:
Why including changes not related to BV scope?
Please refrain about adding changes not strictly related to BV into this PR.
It's already big enough.